src/spreadinterp.cpp

```diff
@@ -218,18 +219,11 @@ void print_subgrid_info(int ndims, BIGINT offset1, BIGINT offset2, BIGINT offset
 */
 template<typename T>
 static FINUFFT_ALWAYS_INLINE T fold_rescale(const T x, const UBIGINT N) noexcept {
-  static constexpr const T x2pi = T(M_1_2PI);
+  static constexpr const T x2pi = T(INV_2PI);
```

Comment:
I think we can just get rid of this variable the name INV_2PI is clear enough

Reply:
Done!